clipper_admin/clipper_admin/deployers/onnx.py

```diff
@@ -137,23 +143,30 @@ def deploy_caffe2_model(
         The number of replicas of the model to create. The number of replicas
         for a model can be changed at any time with
         :py:meth:`clipper.ClipperConnection.set_num_replicas`.
+    onnx_backend : str, optional
+        The provided onnx backend.
```

Comment:
Add the same comment as above about `caffe2` being the only currently supported ONNX backend